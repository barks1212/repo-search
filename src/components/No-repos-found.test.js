import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import React from 'react';
import renderer from 'react-test-renderer';
import NoReposFound from './No-repos-found';

describe('<NoReposFound>', () => {
  it('renders as expected without crashing and matches snapshot', () => {
    const tree = renderer.create(<NoReposFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});