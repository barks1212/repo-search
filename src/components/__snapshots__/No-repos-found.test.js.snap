// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoReposFound> renders as expected without crashing and matches snapshot 1`] = `
<section
  className="noRepo"
>
  <section
    className="container is-fluid"
    id="errorMessage"
  >
    <h2
      className="title"
      id="sorry"
    >
      Sorry!
    </h2>
    <span
      className="sadFace is-size-1"
    >
      <i
        className="far fa-frown"
      />
    </span>
    <h3
      className="subtitle is-size-4"
    >
      No repos found
    </h3>
  </section>
</section>
`;
